src/main/java/frc/robot/commands/DebugValue.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.commands;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
+
+public class DebugValue {
+
+    public static java.lang.Object Get(String DataName) {
+        NetworkTable Table = NetworkTableInstance.getDefault().getTable(DataName);
```

Comment:
Table could be scoped to the class to reduce the repetition. (Yeah, this is a hardcore annoying for literally 19).